resources/views/partials/bootstrap-table.blade.php

```diff
@@ -15,15 +14,17 @@
 <script src="{{ asset('js/extensions/sticky-header/bootstrap-table-sticky-header.js') }}"></script>
 @endif
 
-<script nonce="{{ csrf_token() }}">
+<script src="{{ asset('js/extensions/cookie/bootstrap-table-cookie.js?v=1') }}"></script>
 
-    var $table = $('.snipe-table');
 
+<script nonce="{{ csrf_token() }}">
 
+    var $table = $('.snipe-table');
```

Comment:
I think this bit can be cleaned up - the only place you reference `$table` is within `buildTable()`, the only thing you run in your DOM=ready function is `buildTable` - you should be able to wrap all that together into the body of the DOM-ready function instead.